processing/src/main/java/io/druid/query/aggregation/cardinality/types/DoubleCardinalityAggregatorColumnSelectorStrategy.java

```diff
@@ -31,12 +32,16 @@
   @Override
```

Comment:
Please add a javadoc to `DoubleCardinalityAggregatorColumnSelectorStrategy` stating that "if performance of this class appears to be a bottleneck for somebody, one simple way to improve it is to split it into two different classes, one that is used when {@link NullHandling.useDefaultValuesForNull()} is false, and one - when it's true, moving this computation out of the tight loop".

Reply:
done.